src/test/blockencodings_tests.cpp

```diff
@@ -51,8 +51,8 @@ static CBlock BuildBlockTestCase() {
 }
 
 // Number of shared use_counts we expect for a tx we haven't touched
-// (block + mempool + our copy from the GetSharedTx call)
-constexpr long SHARED_TX_OFFSET{3};
+// (block + mempool entry + mempool txns_randomized + our copy from the GetSharedTx call)
```

Comment:
In b0dda173e1784f177152349ff7002345f054f888 [refactor] rewrite vTxHashes as a vector of CTransactionRef 
`txns_randomized` is introduced in next commit d7eec401ba7c7b4c8fc58e39e358ba4518da97d7

Does this belongs to this commit?